demo/export_presets.cfg

```diff
@@ -2,7 +2,7 @@
 
 name="Meta Quest"
 platform="Android"
-runnable=true
+runnable=false
```

Comment:
We'll need to decide which Android preset we really want to be the runnable one in the repo :-)

But this is fine for now